mojoal/PSPBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: stdgregwar github.com/stdgregwar
+pkgname=mojoal
+pkgver=1.0.0
+pkgrel=1
+pkgdesc="MojoAL is a full OpenAL 1.1 implementation, written in C, in a single source file."
+arch=('mips')
+url="https://icculus.org/mojoAL/"
+license=('ZLIB')
+depends=('sdl2')
+makedepends=()
+optdepends=()
+provides=("openal")
+source=("git+https://github.com/icculus/mojoAL" "psp.patch")
+sha256sums=('SKIP' 'SKIP')
+
+prepare() {
+    cd mojoAL
+    patch -p1 < ../psp.patch
+}
+
+build() {
+    cd mojoAL
+    mkdir pspbuild
+    cd pspbuild
+    psp-cmake .. -DCMAKE_INSTALL_PREFIX=$PSPDEV/psp
+    make
+}
+
+package() {
+    cd mojoAL
+    
+    mkdir -m 755 -p "$pkgdir/psp/lib" 
+    install -m 644 "pspbuild/libmojoal.a" "$pkgdir/psp/lib"
+
+    mkdir -m 755 -p "$pkgdir/psp/include/AL"
```

Comment:
Better give this one its own include directory in case we package OpenAL later. Could be ``mojoal/AL``

Reply:
I think those headers are the official Creative headers for OpenAL anyway. I was more thinking that mojoal and openal-soft would be competing packages providing the target "openal" no ?

Reply:
That is possible, but not preferred. Does MojoAL offer any benefit over OpenAL-Soft? Or could be remove MojoAL once OpenAL-Soft works again?

Reply:
MojoAL is simpler to build and sits on SDL2, that means you cannot use it without SDL2. I do not even know if it is more performant, especially on the psp. I would be happy to profile this once I get things running.